tests/conftest.py

```diff
@@ -51,6 +51,10 @@ def get_mocked_socket(*args, **kwargs):
         mock_socket = MockSocket(*args, **kwargs)
         return mock_socket
 
+    MockedOtherMessage.next_qubit_id = 1
+
+    CQCVariable._next_ref_id = 0
+
     monkeypatch.setattr(socket, "socket", get_mocked_socket)
```

Comment:
This may not be the right place for these lines, but it is important that these variables are reset in the fixture. Please tell me where I should put these lines if this is the wrong place.